resources/application-connector/charts/connection-token-handler/templates/tests/test.yaml

```diff
@@ -2,10 +2,13 @@
 apiVersion: "testing.kyma-project.io/v1alpha1"
 kind: TestDefinition
 metadata:
-  name: {{ .Chart.Name }}-tests
+  name: application-{{ .Chart.Name }}
```

Comment:
Shouldn't it be just the `name: {{ .Chart.Name }}`?

Reply:
I thought it might be nice to prefix it with the main chart name like for the "core" charts. But application-connector-connection-xxx-xx" sounded strange.
But if you like I can change it

Reply:
I do not have a particular preference, the thing is that it is just different than in the other charts under the `application-connector` like here https://github.com/kyma-project/kyma/pull/4958/files/7d32049154b5a8cc3de8ae306ef60e439c30a7f8#diff-ab13db9dc84bd2067c299d2e4e071abeR6 or here https://github.com/kyma-project/kyma/pull/4958/files/7d32049154b5a8cc3de8ae306ef60e439c30a7f8#diff-7b858abc7c31b5ba35e6505af3f45c1fR6 so I think we should stick to one convention.

Reply:
In the references the sub-charts starting with "application-xx" so I thought to better not call it "application-application-xxx"

Either we use the plain sub chart name or we prefix it all with "application-connector"

Reply:
Fair point.
Then I would suggest using a plain sub chart name.

Reply:
fixed